docs/tempo/website/setup/linux.md

```diff
@@ -0,0 +1,299 @@
+---
+title: Deploy on Linux
+menuTitle: Deploy on Linux
+description: Learn how to deploy Tempo on Linux
+weight: 100
+draft: true
+---
+
+<!-- This page was migrated from GET and needs to have the repo and file locations updated for Tempo. -->
+
+# Deploy on Linux
+
+This guide provides a step-by-step process for installing Tempo on Linux.
+It assumes you have access to a Linux machine and the permissions required to deploy a service with network and filesystem access.
+At the end of this guide, you will have deployed a single Tempo instance on a single node.
+
+## Before you begin
+
+To follow this guide, you need:
+
+- A running Grafana instance (see [installation instructions](https://grafana.com/docs/grafana/latest/setup-grafana/installation/))
+- An Amazon S3 compatible object store
+<!-- - Git and Docker installed to run the TNS app -->
+
+### System prerequisites
+
+This configuration is an example you can use as a starting point.
+You may need to have more resources for your system than the minimum specifications listed below.
+Additional adjustments will be necessary for a production environment.
+
+You must have the permissions required to deploy a service with a network and file system access.
+
+Your Linux system should have at least:
+
+- 4 CPUs
+- 16 GB of memory
+
+## Setup an object storage bucket
+
+Tempo uses object storage as the backend for its trace storage.
+It also uses object storage for storing various administrative credentials and data related to the state of the system.
+
+Tempo supports using the local filesystem as the backend for trace storage as well.
+This is not recommended for production deployments and is not supported for storing admin credentials, this guide focuses on setup with an object storage.
+
+This example uses [Amazon S3](https://docs.aws.amazon.com/AmazonS3/latest/userguide/Welcome.html) on the AWS `us-east-1` region as your object store.
+If you plan on using a different region or object storage service, update the storage fields in the configuration file below. Currently, the supported object storage backends are AWS S3, other S3-compliant object stores, and Google Cloud’s GCS.
+
+After you have provisioned an object storage backend, create two buckets: `grafana-traces-admin` and `grafana-traces-data`.
+These buckets will be referenced in the configuration file of this guide.
+You may need to alter the bucket names to be globally unique.
+
+Consider adding a prefix for your organization to the bucket, for example, `myorg-grafana-traces-admin` and `myorg-grafana-traces-data`, and then replacing the names in the rest of these instructions with those bucket names.
+
+## Install Tempo
```

Comment:
A few `enterprise-traces` references here I expect might need to become `tempo` references.

Reply:
Yes, this is the linux.md page that is still being updated. It was the one page that we didn't quite finish this round.